docs/output/targeted.md

```diff
@@ -0,0 +1,238 @@
+---
+order: 1
```

Comment:
This is shown in the markdown file.

Reply:
That's a feature from the new website :)